client/state/happychat/selectors.js

```diff
@@ -41,6 +43,12 @@ export const HAPPYCHAT_CHAT_STATUS_PENDING = 'pending';
  */
 export const getGroups = ( state, siteId ) => {
 	const groups = [];
+
+	if ( isEnabled( 'jetpack/happychat' ) && getSectionName( state ) === 'jetpackConnect' ) {
+		groups.push( HAPPYCHAT_GROUP_JPOP );
+		return groups;
```

Comment:
We could just return `[ HAPPYCHAT_GROUP_JPOP ]` here and have `const groups = [];` after this new block, but it is absolutely fine as it is.

Reply:
Sounds good either way, but I'm leaving it as-is in case we decide to introduce some logical changes (like having a default group that handles all types of chats for exampe) and allow for multiple groups to be stacked there.